eclipse-scout-core/src/planner/Planner.ts

```diff
@@ -344,22 +344,25 @@ export class Planner extends Widget implements PlannerModel {
 
     this.startRow = selectedResource;
     this.lastRow = this.startRow;
-    this._select(true);
+    this._select();
 
     // add event handlers
-    this._resourceTitleMousemoveHandler = this._onResourceTitleMousemove.bind(this, event);
-    this._$body.document()
-      .on('mousemove', this._resourceTitleMousemoveHandler)
-      .one('mouseup', this._onDocumentMouseUp.bind(this));
+    this._removeMouseMoveHandlers();
+    if (this.multiSelect) {
+      this._resourceTitleMousemoveHandler = this._onResourceTitleMousemove.bind(this);
```

Comment:
Wouldn't it be easier if these event handlers were initialized once in the constructor? This would also simplify _removeMouseMoveHandlers().

Reply:
Yes, but _onCellMousemove is bound to the mouse down event so it needs to be bound on every mouse down. To keep it consistent I decided to use the same pattern for the other handlers as well.